ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -102,6 +102,49 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlPostWithSession(SteamCommunityURL, request, data, referer).ConfigureAwait(false);
 		}
 
+		internal async Task AcceptDigitalGiftCards() {
+			const string requestGifts = SteamStoreURL + "/gifts";
+			HtmlDocument response = (await WebLimitRequest(SteamStoreURL, async () => await WebBrowser.UrlGetToHtmlDocument(requestGifts).ConfigureAwait(false)).ConfigureAwait(false))?.Content;
+			if (response == null) {
+				Bot.ArchiLogger.LogNullError(nameof(response));
+				return;
+			}
+
+			const string requestRedeemGift = "/gifts/0/resolvegiftcard";
+			HtmlNodeCollection nodes = response.DocumentNode.SelectNodes("//div[starts-with(@id,'pending_gift_')]");
+			foreach (string strGID in nodes.Where(node => node?.ChildNodes?.Any(giftNode => giftNode?.Attributes["class"]?.Value?.StartsWith("pending_giftcard") == true) == true).Select(node => node.Attributes["id"]?.Value?.Substring(13))) {
```

Comment:
Navigating further with LINQ over HTML is inappropriate, this is what xpath should be used for. I'm not sure how exactly the html looks like (waiting above for sample), but even blindly I can state that `//div[starts-with(@id,'pending_gift_')]/div[starts-with(@class,'pending_giftcard')]/@id` looks 10x better. Also you have excessive number of nullable attributes here, you do just go around and add `?` to every single object, for example `node` can't be `null` in any possible case, so I have no clue why you mark it as nullable.